File: CyberRange/CyberRangeProject/Models/Task/TaskModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace CyberRangeProject.Models.Task {

	public class TaskModel {

		[Key]
		public int TaskId { get; set; }

		public string Icon { get; set; }
		public string Name { get; set; }
		public string Description { get; set; }
		public int Duration { get; set; }
		public string State { get; set; }
		public int MaxPoints { get; set; }
		public bool RandomFlag { get; set; }
		public string Flag { get; set; }
		public string TemplateName { get; set; }
	}
}

File: CyberRange/CyberRangeAngular/src/app/admin/task/task-model.ts
export class TaskModel {
    public constructor(init?: Partial<TaskModel>) {
        Object.assign(this, init);
    }
    TaskId: number;
    Icon: string;
    Name: string;
    Description: string;
    Duration: number;
    State: string;
    MaxPoints: number;
    RandomFlag: boolean;
    Flag: string;
    TemplateName: string; 
}